index.bs

```diff
@@ -7454,15 +7428,22 @@ In effect, where <var ignore>x</var> is a Number value,
 </div>
 
 
-<h4 id="es-float">float</h4>
+<h4 id="es-decimal-types">Decimal types</h4>
+
+<h5 oldids="es-unrestricted-float" id="es-float">float</h5 id="es-unrestricted-float">
```

Comment:
Closing tags should not have attributes.

Reply:
That’s caused by a bug in **Visual Studio Code**’s new [**HTML mirror cursor**](https://code.visualstudio.com/updates/v1_41#_html-mirror-cursor) feature: <https://github.com/microsoft/vscode/issues/87067>.

It will hopefully be fixed by <https://github.com/microsoft/vscode/issues/88424#issuecomment-600840619>.